build.gradle

```diff
@@ -2,7 +2,7 @@ plugins {
   id "idea"
   id "com.github.mxenabled.binks" version "0.0.1"
   id "com.github.mxenabled.vogue" version "1.0.1"
-  id "com.github.mxenabled.coppuccino" version "3.1.0" apply false
+  id "com.github.mxenabled.coppuccino" version "enable_unused_import_javadoc-SNAPSHOT" apply false
```

Comment:
Is this version change intentional?

Reply:
Yes. I'll chance it back when the new coppuccino is approved and released. https://github.com/mxenabled/coppuccino/pull/23